### ISSUE TITLE
[QUESTION] - Phone doesn't boot

### ISSUE THREAD

**Issue by TrustedFloppa** [ETCHDEV/Port-Windows-11-Xiaomi-11-Lite-NE#9]
### Question

Hello friend, at first i would like to thank you for this project. however I ran into issue.. It looks like after I rebooted phone as it was stuck at edk2 logo then it shows xiaomi logo and it doesn't show unlocked, just like its locked. I cant get it into fastboot, I flashed install image as boot and neither does recovery or bootloader work.

### Agreement

- [X] I am asking a sensible question.
- [X] I am not trying to waste time.
- [X] There is no issue already describing my problem.

**Comment by TrustedFloppa** [comment]
OK, I fixed it, but here is my question.
1. Why does it forever on the first boot when the rdp command?
2. when I have rdp then I put Phone-XXXXXX in the IP and then connect? or another way?
I got a little confused here...
3. If I want to write drivers for the touch then I base on these files I found in UFS?
![image](https://github.com/ETCHDEV/Port-Windows-11-Xiaomi-11-Lite-NE/assets/85476585/4a8966b9-c6dc-440b-9244-eaf8a5011593)
